source/Rules.txt

```diff
@@ -0,0 +1,107 @@
+**OBJECT OF THE GAME**
+
+Fugitive is a quick 2 player game where one player is the Fugitive moving from hideout to hideout on his escape route, and the other player is the Marshal trying to catch him before he escapes.
+
+The main deck is composed of 43 Hideout cards, which represent places the Fugitive can hide. 
+
+The Fugitive wins by playing cards until he can eventu­ally play the #42 Hideout card to escape the Marshal.
+
+The Marshal wins by identifying all the Fugitive's hide­ outs to find and arrest the Fugitive.
+
+
+
+**SETUP**
+
+The hideout cards are sorted into three decks: 4-14, 15-28, and 29-41. Each deck is shuffled separately.
+
+The fugitive recieves a starting hand. (the marshal has no starting hand). The fugitive starts with the hideout cards numbered 1, 2, 3, and 42.
```

Comment:
This is a bit of a stylistic suggestion, but I believe that receives is spelled incorrectly. Otherwise, thank you for the work on this George! I envision integrating the rules file as either opening the text file or printing it out into the active gameplay. Considering the length, I think opening it as a separate file would be best. Let me know what you think!

Reply:
Thanks for catching that typo! I went ahead and made the change. I think we can create some functionality asking the user if they would like to see the rules, and if we do, we can just open the other file and print it. We can discuss in scrum